common/src/main/java/tachyon/TachyonURI.java

```diff
@@ -41,10 +42,10 @@
   private final URI mUri;
 
   /**
-   * Construct a TachyonURI from a String. Path strings are URIs, but with unescaped elements and
-   * some additional normalization.
+   * Construct a {@code TachyonURI} from a String. Path strings are URIs, but with unescaped
```

Comment:
nit: since you cleaning things up, could you please make sure that all javadoc method-level comments in this file use 3rd person (e.g. Constructs, Resolves, ...) instead of 2nd person (e.g. Construct, Resolve)? thanks